### ISSUE TITLE
Fork-ts-checker-webpack-plugin seems to run twice after change in code 

### ISSUE THREAD

**Issue by janlent1** [TypeStrong/fork-ts-checker-webpack-plugin#742]
## Current behavior
When running in dev-proxy mode, the checker seems to not be aborted but still called again.
That can be seen here:

[0] runtime modules 1.27 MiB 192 modules
[0] javascript modules 3.8 MiB 250 modules
[0] webpack 5.69.0 compiled successfully in 43950 ms
[0] Type-checking in progress...
[0] No errors found.
[0] No errors found.
## Expected behavior

It should only output no errors found once.

## Steps to reproduce the issue

When retriggering the compilation because of some code change, that was sometimes observed, I guess the first typechecking is not yet finished.
## Issue reproduction repository

Not possible, production repo of a company repo.
## Environment
 - **fork-ts-checker-webpack-plugin**: 7.2.6
 - **typescript**: 4.1.5
 - **eslint**: [version from the `package.json`]
 - **webpack**: 5.69.0
 - **os**: Windows and MacOS

**Comment by piotr-oles** [comment]
I will not be able to help you without a reproduction repository :) I have limited time that I can spend on this project and I do all of this for free - didn't earned a cent on this 😃

**Comment by dsagal** [comment]
@janlent1, which editor are you using? If vim, this may be relevant: https://webpack.js.org/configuration/watch/#vim. One fix for this (or similar) issue is to add this to the webpack config:
```
  watchOptions: {
    aggregateTimeout: 250
  }
```

**Comment by janlent1** [comment]
Seems to be fixed with https://github.com/TypeStrong/fork-ts-checker-webpack-plugin/releases/tag/v7.2.13
